package.json

```diff
@@ -1,9 +1,9 @@
 {
 	"name": "meadow",
 	"displayName": "Meadow",
-	"version": "0.3.0",
-	"publisher": "wildernesslabs",
-	"description": "Build and Deploy Meadow applications",
+	"version": "0.18.6",
```

Comment:
@adrianstevens, @jorgedevs I'd like to sync the major version numbers across extensions is possible. Only changing minor version numbers, in the case where we need specific fixes to an extension. This is because they all now use the side-by-side Meadow-CLI checkout, so the basic internal should, theoretically, be the same. Or have I over simplified the issues? Is syncing major version numbers a good idea??